libnmstate/state.py

```diff
@@ -159,6 +217,42 @@ def merge_interfaces(self, other_state):
             dict_update(other_state.interfaces[name], self.interfaces[name])
             self._ifaces_state[name] = other_state.interfaces[name]
 
+    def merge_route_config(self, other):
+        """
+        Merge routes from other to self.
+        If interface does exists in self, create an empty one.
```

Comment:
I do not understand the sentence above and it is also not correct English (`does exist` without `s`) and there needs to be an article (an or the) before `interface`.

Reply:
Changed to:
```txt
If any changed route referring non-exist interface, create the
interface state with name only, in order to trigger the recreating of
the interface profile.
```